src/CalculatorUITestFramework/WinAppDriver.cs

```diff
@@ -69,6 +72,9 @@ public void SetupCalculatorSession(TestContext context)
                 this.CalculatorSession = new WindowsDriver<WindowsElement>(this.windowsDriverService.ServiceUrl, options);
                 this.CalculatorSession.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
                 Assert.IsNotNull(this.CalculatorSession);
+                this.MemoryPanel.ResizeWindowToDiplayMemoryLabel();
+                //Size windowSize = new Size(1200, 1050);
```

Comment:
remove if this isn't needed

Reply:
It is needed, it makes the test cases stable and run faster.  I will remove the commented out test. The change will appear in my next commit